app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name, age, wife=None, husband=None):
+        self.name = name
+        self.age = age
+        if wife is not None:
+            self.wife = wife
+        if husband is not None:
+            self.husband = husband
+        self.__class__.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for person in people:
+        Person(
+            person["name"],
+            person["age"],
+            wife=person["wife"] if "wife" in person else None,
+            husband=person["husband"] if "husband" in person else None
```

Comment:
You added "wife" and "husband" twice. Here and at $23 - $28.
Also we dont need $7 - $10, because we add with Person.people[person['name']].husband which automatly added self.husband or self.wife